### ISSUE TITLE
Port Istio repo to CentOS

### ISSUE THREAD

**Issue by sdake** [istio/istio#922]
Port the istio/istio repository to use CentOS based images.  From initial observation, this mainly effects the mixer, mixer_debug, iptables, and proxy_init containers.  There are two possible solutions:

1. Build ubuntu and centos based versions of these containers
2. Statically link mixs and xtables-multi.  mixs uses many system shared objects which may be lacking static libraries for the various distributions intended for support.  xtables-multi may be an easier static link, assuming RTNETLINK works cross-kernel.

I have patches for option 1 to enable more community involvement.  I'm not sure immediately how to tackle option 2, especially mixs.

**Comment by sdake** [comment]
Note the istio user issue  tracker for this problem is here: https://github.com/istio/issues/issues/56

**Comment by sdake** [comment]
Dependent mixer patch is here: https://github.com/istio/mixer/issues/1347

**Comment by linsun** [comment]
this seems not required for 0.2, moving to 0.3 or future.

**Comment by ldemailly** [comment]
Can we get some specifics about what exactly isn't working

My (maybe naive) understanding of docker is that it doesn't matter what the host OS is

**Comment by sdake** [comment]
@ldemailly sure - I'll work up a gist of master (using the Ubuntu based containers) that shows the problem in more detail.

**Comment by sdake** [comment]
@ldemailly I gave master of Istio a try, and that works now without compiling locally on CentOS.  Prior I was receiving 503 errors, although I believe that was fixed in a different issue.  Compiling go binaries on CentOS and inserting them in the ubuntu based distros did not previously appear to function properly.  I'll re-verify that problem (the developer workflow).

Thanks for not merging blindly and the helpful reviews; still coming up to speed on the codebase.

Cheers,
-steve
